cluster-baseline-settings/akv-secrets-store-csi.yaml

```diff
@@ -203,9 +344,9 @@ spec:
           # PRODUCTION READINESS CHANGE REQUIRED
           # This image should be sourced from a non-public container registry, such as the
           # one deployed along side of this reference implementation.
-          # az acr import --source docker.io/weaveworks/kured:1.4.0 -n <your-acr-instance-name>
+          # az acr import --source quay.io/k8scsi/csi-node-driver-registrar:v1.2.0 -n <your-acr-instance-name>
           # and then set this to
-          # image: <your-acr-instance-name>.azurecr.io/weaveworks/kured:1.4.0
+          # image: <your-acr-instance-name>.azurecr.io/k8scsi/csi-node-driver-registrar:v1.2.0
```

Comment:
Are we pulling out kured:1.4.0 altogether?  If so, then we should update [06-gitops.md](06-gitops.md), [inner-loop-scripts/azcli/cluster-deploy.azcli](inner-loop-scripts/azcli/cluster-deploy.azcli), and possibly remove [cluster-baseline-settings/kured-1.4.0-dockerhub.yaml](cluster-baseline-settings/kured-1.4.0-dockerhub.yaml) - I don't have enough context on the change to know if this makes sense.

Reply:
Nope, that was just me fixing a typo (copy and paste whoops) from a prior PR.